src/Umbraco.Web/Models/RelatedLinkBase.cs

```diff
@@ -14,5 +15,7 @@ public abstract class RelatedLinkBase
         public bool IsInternal { get; set; }
         [JsonProperty("type")]
         public RelatedLinkType Type { get; set; }
+        [JsonProperty("content")]
```

Comment:
Not sure at all an entire `IPublishedContent` can be a json property, it's quite probably not serializable, and certainly _not_ deserializable without special plumbing as it's an interface?